tools/ci_connector_ops/ci_connector_ops/pipelines/commands/groups/connectors.py

```diff
@@ -164,10 +145,16 @@ def test(
         ctx (click.Context): The click context.
     """
     click.secho(f"Will run the test pipeline for the following connectors: {', '.join(ctx.obj['selected_connectors_names'])}.", fg="green")
+    if ctx.obj["selected_connectors_and_files"]:
+        update_global_commit_status_check_for_tests(ctx.obj, "pending")
+    else:
+        click.secho("No connector were selected for testing.", fg="yellow")
+        update_global_commit_status_check_for_tests(ctx.obj, "success")
+        return True
 
     connectors_tests_contexts = [
         ConnectorContext(
-            pipeline_name="Test",
+            pipeline_name=f"Testing connector {connector.technical_name}",
```

Comment:
This will end up being the name of the connector specific status check.

Reply:
dynamic name, nice!